core/src/replay_stage.rs

```diff
@@ -1659,12 +1671,16 @@ impl ReplayStage {
         replay_vote_sender: &ReplayVoteSender,
         transaction_cost_metrics_sender: Option<&TransactionCostMetricsSender>,
         verify_recyclers: &VerifyRecyclers,
+        handle: &BankTransactionExecutorHandle,
     ) -> result::Result<usize, BlockstoreProcessorError> {
         let tx_count_before = bank_progress.replay_progress.num_txs;
         // All errors must lead to marking the slot as dead, otherwise,
         // the `check_slot_agrees_with_cluster()` called by `replay_active_banks()`
         // will break!
-        blockstore_processor::confirm_slot(
+
+        // loop over until no more entries to process
+        // TODO (LB): probably wanna add a time limit here
+        while blockstore_processor::confirm_slot(
```

Comment:
Just to clarify, this will now attempt to replay all available entries for a bank vs. jumping back to call-site after replaying one set of entries?

Reply:
yeah